client/contracts/bindings/consensus/consensus.go

```diff
@@ -7,6 +7,10 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/kowala-tech/kcoin/client/common/kns"
+
```

Comment:
spaces

Reply:
Isn't that correct?

Reply:
I mean the extra spaces :D between imports.

Reply:
done, there were some additonal enters.